temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/IntervalPickerRow.kt

```diff
@@ -73,13 +74,14 @@ fun IntervalPickerRow(
             textColor = if (validInput) White else UI.colors.pureInverse,
             hint = "0"
         ) {
+            val filteredText = if (it.text.length <= 3) it.text else it.text.take(3)
```

Comment:
Extract `3` as a constant (Detekt told you probably) and `it.text.take(3)` is sufficient  - no need for an `if`.

Additionally change the constant to 5 to give more room for custom stuff

Reply:
@ILIYANGERMANOV In my opinion if we are limiting the entered amount then its no need for that width attribute or any other changes to the UI we can just it.text.take(3) make this 5, as you suggested . Additionally i have a question about where should i declare the constant . There are multiple constants file in the application or should i declare in the same file itself ? 
![image](https://github.com/user-attachments/assets/0e23f9f0-4589-4928-ae6c-11ac3e0a67c5)

Reply:
@akashs056 in the same file where you make the change. At the top of the file

Reply:
it.text.take(RepeatIntervalCharLimit)